src/features/commands/watch.js

```diff
@@ -0,0 +1,169 @@
+                        interaction.reply({
+                            content: 'Allow Direct Messages from server members in this server for this feature to work.',
+                            ephemeral: true,
+                        })
+                    }
+                });
+                break;
+
+            case "remove":
+                const keywordList = new MessageActionRow();
+                const Buttons = new MessageActionRow()
+                    .addComponents(
+                        new MessageButton()
+                            .setCustomId(`removeAll`)
+                            .setLabel(`Remove all keywords`)
+                            .setStyle(`DANGER`),
+                        new MessageButton()
+                            .setCustomId(`doNothing`)
+                            .setLabel(`Don't remove anything`)
+                            .setStyle(`SECONDARY`)
+                    )
+                db.getWatchedKeywords(interaction.user.id, interaction.guild.id).then(keywords => {
+                    if (keywords.length === 0) {
+                        interaction.reply({
+                            content: `You aren't tracking any keywords for this server. Track words by using the \`/watch\` command!`,
+                            ephemeral: true,
+                        })
+                    } else {
+                        try {
+                            const list = keywords[0].watchedWords
+                            let components = []
+                            for (i = 0; i < list.length; i++) {
+                                const value = {
+                                    label: `${list[i]}`,
+                                    description: `Remove ${list[i]} from watched keywords.`,
+                                    value: `${list[i]}`
+                                }
+                                components.push(value)
+                            }
+                            keywordList.addComponents(
+                                new MessageSelectMenu()
+                                    .setCustomId('keywords')
+                                    .setPlaceholder('Nothing Selected')
+                                    .setMinValues(1)
+                                    .setMaxValues(list.length)
+                                    .addOptions(components),
+                            );
+                            interaction.reply({
+                                content: `Select the keywords you want to remove.`,
+                                components: [keywordList, Buttons],
+                                ephemeral: true,
+                            })
+                            return
+                        } catch (error) {
+                            console.log(error)
+                            interaction.reply({
+                                content: `Something went wrong:\n${error}`,
```

Comment:
I'd prefer not to show unsanitized errors to the users.
```suggestion
                                content: 'Something went wrong.',
```